### ISSUE TITLE
[Keyboard Navigation - Azure Bot Services - Test in Webchat]: Keyboard focus is not visible properly when it reaches on the 'Attach' control

### ISSUE THREAD

**Issue by kattepalli** [microsoft/BotFramework-WebChat#3873]
**User Experience:**
If users navigate on screen and Keyboard focus is not visible properly when it reaches on the 'Attach' control, so it would be difficult for users to navigate on page and complete their task.

**Repro Steps:​**
Open https://ms.portal.azure.com/?microsoft_azure_marketplace_ItemHideKey=BotWithComposer&Microsoft_Azure_BotService=friendName#create/hub using v- in which access provided by product team.
Navigate to search box and search "Azure Bot" and  select "Azurebot services".
Navigate to Bot services window and select already created Bot.
Navigate to "Test in Web Chat" list item under azure bot window and select it.
Navigate all the controls of "Test in Web Chat" blade under Azure bot window.
Use accessibility insights for web and Verify the issue.

**Actual Result:**
The keyboard focus is not visible properly when it reaches the 'Attach' control.

**Expected Result:**
Keyboard focus should be visible properly when it reaches the 'Attach' control. 

**MAS Reference:**
[MAS 2.4.7 – Focus Visible](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7b5ED5CC1A-4764-40CD-8F53-C8107BA0D5A2%7d&file=MAS%202.4.7%20%E2%80%93%20Focus%20Visible%20.docx&wdLOR=c651B304E-2BC4-4A72-B13D-C09B41B14249&action=default&mobileredirect=true)

**Test Environment:**
OS: Win 10 20H2
Build: OS Build 19042.867
Platform: Edge -Version 91.0.852.0 (Official build) dev (64-bit), Chrome- Version 89.0.4389.114 (Official Build) (64-bit)
Tool: Accessibility Insights for Web
#URL: [Azure bot services](https://ms.portal.azure.com/?microsoft_azure_marketplace_ItemHideKey=BotWithComposer&Microsoft_Azure_BotService=friendName#create/hub)

**Comment by corinagum** [comment]
Could you please provide a screenshot of the behavior you are indicating?

**Comment by kattepalli** [comment]
Please find below a screenshot 
![MAS2 4 7_focus not visible properly1](https://user-images.githubusercontent.com/30458420/117948504-cfbbf000-b32e-11eb-8261-46bedff12432.png)

**Comment by corinagum** [comment]
This is behavior that's been verified with our audit team - the focus indicator change is in the color of the paperclip/send button, which match color contrast requirements. There is no plan for a visual update at this time.
